css/main.css

```diff
@@ -272,11 +272,307 @@ header .container button {
     gap: 20px;
     padding: 36px 0 46px;
 
+
     @media screen and (max-width: 1160px) {
         overflow-x: auto;
     }
 }
 
+.companies .container .companies-inner .i1 {
+    animation-name: companies-animation1;
+    animation-duration: 10s;
+    animation-iteration-count: infinite;
+}
+
+@keyframes companies-animation1 {
+
+    10% {
+        transform: translateX(-124%);
+    }
+
+    11% {
+        transform: translate(-124%, 300%);
+    }
+
+    12% {
+        transform: translate(744%, 300%);
+    }
+
+    13% {
+        transform: translateX(744%);
+    }
+
+    20% {
+        transform: translateX(620%);
+    }
+
+    30% {
+        transform: translateX(496%);
+    }
+
+    40% {
+        transform: translateX(372%);
+    }
+
+    50% {
+        transform: translateX(248%);
+    }
+
+    60% {
+        transform: translateX(124%);
+    }
+
+    70% {
+        transform: none;
+    }
+
+}
+
+.companies .container .companies-inner .i2 {
```

Comment:
i1, i2, i3 - this is the bad practice of adding name to elements. i1 -> the worst name to element